storm-client/src/jvm/org/apache/storm/pacemaker/PacemakerClient.java

```diff
@@ -153,13 +154,13 @@ public String secretKey() {
     }
 
     public HBMessage send(HBMessage m) throws PacemakerConnectionException, InterruptedException {
-        LOG.debug("Sending message: {}", m.toString());
+        LOG.debug("Sending pacemaker message to {}: {}", host, m.toString());
```

Comment:
nit: can we drop the `.toString()` for m? The log command should do it for you, if it is needed.